src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AotTests.cs

```diff
@@ -144,9 +144,7 @@ public void BuildBasicApplicationReleaseProfiledAotWithoutDefaultProfile ()
 				AotAssemblies = true,
 				PackageName = "com.xamarin.buildaotappwithspecialchars",
 			};
-			if (!Builder.UseDotNet) {
-				proj.BundleAssemblies = true;
-			}
+
```

Comment:
I think there is an assert below this that pivots based on BundleAssemblies that may need updating.